src/services/national-insurance.js

```diff
@@ -25,7 +25,28 @@ const bandsOnDate = (date) => {
 };
 
 // TODO this should do more than return the number it's given
-const slice = R.curry((floor, ceiling, num) => num);
+const slice = R.curry((floor, ceiling, num) => {
```

Comment:
this isn't the cleanest solution but you have achieved the tests passing and the commentary on your logic is spot-on